webextensions/api/browserAction.json

```diff
@@ -204,7 +204,7 @@
                 "version_added": "57"
               },
               "firefox_android": {
-                "version_added": false
+                "version_added": "57"
```

Comment:
I haven't tested this, but are you sure `openPopup` is supported in Android? According to http://searchfox.org/mozilla-central/source/mobile/android/components/extensions/schemas/browser_action.json#412 it is not.